protobuf/proto/agent.proto

```diff
@@ -41,6 +41,8 @@ message AgentCommand {
   oneof command {
     VersionDeployRequest deploy = 1;
     ContainerStatusRequest containerStatus = 2;
+    ContainerDeleteRequest containerDelete = 3;
+    DeployRequestCore deployCore = 4;
```

Comment:
Use of the word Legacy instead of Core? What do you think? Core might not say anything to the reader.